README.md

```diff
@@ -81,10 +81,6 @@ Observability is collection of plugins and applications that let you visualize d
 [bug-link]: https://github.com/opensearch-project/observability/issues?q=is%3Aopen+is%3Aissue+label%3Abug+
 [enhancement-badge]: https://img.shields.io/github/issues/opensearch-project/observability/enhancement.svg
 [enhancement-link]: https://github.com/opensearch-project/observability/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement+
-<<<<<<< HEAD
```

Comment:
We should probably remove these conflict flags

Reply:
These changes are made to remove the conflicts...

Reply:
Ah. yes you are correct.